### ISSUE TITLE
Unusable SHOW CREATE VIEW when Hive view isn't compatible with Presto syntax

### ISSUE THREAD

**Issue by ebyhr** [trinodb/trino#3693]
Currently, it's experimental feature though.
```sql
presto> show create view hive.default.test_hive;
              Create View
---------------------------------------
 CREATE VIEW hive.default.test_hive AS
 SELECT translate('a', 'a', 'b')
```

```sql
presto> CREATE VIEW hive.default.test_hive AS
     ->  SELECT translate('a', 'a', 'b');
Query 20200511_121510_00009_8jtau failed: line 2:9: Function 'translate' not registered
CREATE VIEW hive.default.test_hive AS
 SELECT translate('a', 'a', 'b')
```

**Comment by kokosing** [comment]
Is it still an issue once we have Coral integration?

**Comment by ebyhr** [comment]
We get an exception in case of unsupported Hive view now. Let me close this issue.

**Comment by findepi** [comment]
@ebyhr do we have a test for that?

Also, is it actually the correct behavior? `SHOW CREATE VIEW` should reveal the view _persisted_ definition as faithfully as possible, and so should not show the translated (transient) result.

**Comment by kokosing** [comment]
> SHOW CREATE VIEW should reveal the view persisted definition as faithfully as possible

That could be surprising for users to see syntax not compatible with Trino. Also if it is the case the with need to negate the issue tittle and description, as this is not a bug but feature to which we introduced a regression.

As I understand, end users might not even know who created given view and where. And if they want to create another based on this one they will be unable to do it as syntax is different.

Maybe Hive views should not be views in Trino, but tables?

**Comment by findepi** [comment]
> That could be surprising for users to see syntax not compatible with Trino. 

I agree, the SHOW CREATE VIEW should return something that would create said view when run _in Trino_ (if successful).

This does not mean, however, that showing the (current) result of translation is the right behavior.

cc @losipiuk @phd3

**Comment by imitevm** [comment]
Is there a workaround for extracting view DDLs for Trino? I'm migrating from Impala to Trino and now the SHOW CREATE VIEW produces insanely unreadable statements. E.g. nested anonymous joins which is nothing like the original statement used to create the view in first place.
To be clear on my case: for a view created with Trino statement SHOW CREATE VIEW from Impala would produce meaningful result but the outcome from the same statement against Trino would be malformed.
